mqtt/mqtt-bridge/src/controller.rs

```diff
@@ -101,7 +104,7 @@ impl BridgeController {
         };
 
         let updates = async move {
-            while let Some(update) = self.updates_receiver.recv().await {
```

Comment:
recv() call was busy?

Reply:
Denis suggested in the original PR to use next